### ISSUE TITLE
onContext not sending any video data with being a panel

### ISSUE THREAD

**Issue by BarryCarlyon** [twitchdev/issues#284]
**Brief description**

onContext is very "dry"/lacking details

**How to reproduce**

https://gist.github.com/BarryCarlyon/71c15dc7f82f28ef5c59064be74e065a

See also #233 

**Expected behavior**

Hydrated data in the onContext callback

**Screenshots**

Left, 3 months ago.
Right, right now.

Same code

![image](https://user-images.githubusercontent.com/20999/103809873-1ed8d800-5052-11eb-8a46-41fd5e7c37e7.png)

**Comment by lleadbet** [comment]
Ticketed internally as ANC-2708.

**Comment by rupture-ttv** [comment]
I'm currently writing an extension and nearly ready to submit but this issue seems to be holding me up.  My extension is game specific and one of the settings the streamer can set is if the extensions functionality should be available
* All the time
* Only when playing the game
* Off

However I am seeing that whenever the stream is started and the game is left the same as it was from previous streams the onContext call always returns game: ""  I left the stream on for over 30 minutes and even new browsers on different networks were reporting the same data in onContext and the game was just an empty string.

However I the broadcaster changed games while live onContext fired the callback with the new game.  Then switching back to the previous game fired it again with the original game which would then activate my extension when the settings were set to "Only when playing the game"

It's sad that I am here 2 years later facing the same issue.  I'm not sure how to go about handling this.  I guess I can treat an empty string as being the correct game and only treat strings that are not empty as being some other game if they don't match the game my extension is for.

Basically "empty string" or "the game" = THE GAME
else NOT THE GAME.

Any better solution?
